app/users/tests/test_views.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.test import RequestFactory, TestCase
+from django.urls import reverse
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+from rest_framework import status
+
+from app.users.views import PasswordReset, ResetPasswordAPI
+
+User = get_user_model()
+
+
+class PasswordResetTest(TestCase):
+    def setUp(self) -> None:
+        self.factory = RequestFactory()
+        self.user = User.objects.create_user(
+            username="chrisnemwel",
+            email="chrisnemwel@gmail.com",
+            password="secretpassword",
+        )
+
+    def test_password_reset_email(self) -> None:
+        url = reverse("request-password-reset")
+        factory = RequestFactory()
```

Comment:
Haven't you already created a factory for each request in the setUp method

Reply:
What if I remove the create factory in the setUp method , Is it proper to create a factory for each request